web/controller/components/home/ControllerHome.js

```diff
@@ -61,18 +64,35 @@ export function ControllerHome() {
           <ControllerHomeCard
             text={"LIC papier présenté"}
             icon={"fr-icon-draft-line fr-icon--lg"}
+            onClick={() =>
+              setModal({ isOpen: true, parcours: "d'un LIC papier" })
+            }
           />
         </Grid>
         <Grid item xs={12} sm={4}>
           <ControllerHomeCard
             text={"Pas de LIC à bord"}
             icon={"fr-icon-alarm-warning-line fr-icon--lg"}
+            onClick={() =>
+              setModal({ isOpen: true, parcours: '"Pas de LIC à bord"' })
+            }
           />
         </Grid>
       </Grid>
       <a className={classNames(classes.noLicLink, "fr-link")} href="#">
         Un horaire de service est présenté ?
       </a>
-    </Container>
+    </Container>,
+    <Modal
+      key={2}
```

Comment:
```suggestion
      key={3}
```
:orange_circle: Pour éviter une erreur dans la console.

Reply:
J'ai plutôt modifié la valeur du component précédent pour que les chiffres se suivent.